articles/azure-monitor/app/tutorial-custom-metrics.md

```diff
@@ -0,0 +1,370 @@
+
+5. Select **Publish** to promote the new code to the Azure App Service.
+
+    ![The AzureCafe publish profile displays with the Publish button highlighted.](./media/asp-net-core/publish_profile.png "Publish profile")
+
+6. Once the publish has succeeded, a new browser window opens to the Azure Cafe web application.
+
+    ![The Azure Cafe web application displays.](./media/asp-net-core/azure_cafe_index.png "Azure Cafe web application")
+
+7. Perform various activities in the web application to generate some telemetry.
+
+   1. Select **Details** next to a Cafe to view its menu and reviews.
+
+        ![A portion of the Azure Cafe list displays with the Details button highlighted.](./media/asp-net-core/cafe_details_button.png "Azure Cafe Details")
+
+   2. On the Cafe screen, select the **Reviews** tab to view and add reviews. Select the **Add review** button to add a review.
+
+        ![The Cafe details screen displays with the Add review button highlighted.](./media/asp-net-core/cafe_add_review_button.png "Add review")
+
+   3. On the Create a review dialog, enter a name, rating, comments, and upload a photo for the review. Once completed, select **Add review**.
+
+        ![The Create a review dialog displays.](./media/asp-net-core/create_a_review_dialog.png "Create a review")
+
+   4. Repeat adding reviews as desired to generate additional telemetry.
+
+### View logs in Application Insights
+
+1. Go to the **Application Insights** resource in the [Azure Portal](https://portal.azure.com).
+
+    ![A resource group displays with the Application Insights resource highlighted.](./media/asp-net-core/application_insights_resource_group.png "Resource Group")
+
+2. From the left menu of the Application Insights resource, select **Logs** from beneath the **Monitoring** section. In the **Tables** pane, double-click on the **customMetrics** table, located under the **Application Insights** tree. Modify the query to retrieve metrics for the **ReviewPerformed** custom named metric as follows, then select **Run** to filter the results.
+
+    ```kql
+    customMetrics 
+    | where name == "ReviewPerformed"
+    ```
+
+3. Observe the results display the rating value present in the Review and the aggregated values.
+
+## Multi-dimensional metrics
+
+The examples in the previous section show zero-dimensional metrics. Metrics can also be multi-dimensional. We currently support up to 10 dimensions.
+
+By default multi-dimensional metrics within the Metric explorer experience aren't turned on in Application Insights resources.
+
+### Enable multi-dimensional metrics
+
+To enable multi-dimensional metrics for an Application Insights resource, Select **Usage and estimated costs** > **Custom Metrics** > **Send custom metrics to Azure Metric Store (With dimensions)** > **OK**.
+
+Once you have made that change and send new multi-dimensional telemetry, you'll be able to **Apply splitting**.
```

Comment:
Also warn that this is a preview feature and additional billing may apply in the future.

Reply:
Fixed.